benchmark/rnd.sh

```diff
@@ -0,0 +1,8 @@
+# export WANDB_ENTITY=openrlbenchmark
+
+poetry install -E envpool
+xvfb-run -a python -m cleanrl_utils.benchmark \
+    --env-ids MontezumaRevenge-v5 \
+    --command "poetry run python cleanrl/ppo_atari_envpool.py --total-timesteps 2000000000 --ent-coef 0.001 --num-envs 128 --gamma 0.999 --int-gamma 0.99 --learning-rate 0.0001 --track --capture-video" \
+    --num-seeds 3 \
```

Comment:
If we only used 1 random seed it should be 1 here, too.  The command was wrong, it should be `ppo_rnd_envpool.py`